__init__.py

```diff
@@ -18,22 +18,19 @@
 import os
 import tempfile
 from lingua_franca.format import get_date_strings
-from mycroft.skills.api import SkillApi
-from mycroft.skills.core import (MycroftSkill, intent_file_handler,
-                                 resting_screen_handler)
-from mycroft_bus_client import Message
-from ovos_skills_manager.utils import get_skills_examples
+from ovos_utils.skills.api import SkillApi
+from ovos_workshop.skills.ovos import OVOSSkill
+from ovos_workshop.decorators import intent_handler, resting_screen_handler
+from ovos_bus_client import Message
 from ovos_utils import classproperty
 from ovos_utils.log import LOG
 from ovos_utils.process_utils import RuntimeRequirements
 
 from .skill import (DashboardHandler, CardGenerator)
 
 
-class OVOSHomescreenSkill(MycroftSkill):
-    # The constructor of the skill, which calls MycroftSkill's constructor
-    def __init__(self):
-        super(OVOSHomescreenSkill, self).__init__(name="OVOSHomescreen")
+class OVOSHomescreenSkill(OVOSSkill):
```

Comment:
I think this is supposed to extend `IdleDisplaySkill` based on the latest GUI module changes

Reply:
i just wanted to update imports, can do that in a follow up PR as this one is already growing in scope and i've already spent way more than i intended working on this just to get rid of a couple logs....